src/main/views/webContentEvents.js

```diff
@@ -136,6 +136,12 @@ const generateNewWindowListener = (getServersFunction, spellcheck) => {
             return;
         }
 
+        // Image proxy case
+        if (parsedURL.pathname.match(/^\/api\/v[3-4]\/image/)) {
```

Comment:
Strictly speaking, this API never existed in API v3, but we could always leave it for consistency